aws/extensions/computetask_awswin_cwlog/extension.ftl

```diff
@@ -51,9 +61,9 @@
             [#local logContent +=
                 [
                     r'{',
-                    r'    "file_path": "' + windows_path_converter(logFileDetails.FilePath) + '",',
+                    r'    "file_path": "' + (logFileDetails.FilePath)?replace(r"\",r"\\") + '",',
```

Comment:
I think the JSON output handling should cover the conversion and escaping. Was it not?

Reply:
Covered in 4a9fa5f